### ISSUE TITLE
Not treating comments as tags in code blocks inside callouts

### ISSUE THREAD

**Issue by claremacrae** [damiankorcz/Prism-Theme#55]
Something curious about the new tag pills in code blocks inside callouts:

Obsdian 0.14.9
Prism: Version: v2.1.0

If I have a code block, such as for tasks, the commented lines (beginning with `#`) are displayed correctly:

This is the code:

````markdown
```tasks
#group by folder
description contains stuff
#commented out line
limit 5
```
````

This is in source mode, and is as expected:

![image](https://user-images.githubusercontent.com/4840096/167303889-0592bf4e-2aa1-42aa-9a1d-faaf6ca03dee.png)

If I wrap that tasks block in a callout, I see this:

![image](https://user-images.githubusercontent.com/4840096/167303921-a97037d9-f593-4f01-a90e-82b1b036c46d.png)

Might it be possible, at some point please, for `#` characters not to be treated as tags if they are in code blocks, just because the code block is inside a callout?

**Comment by claremacrae** [comment]
Note to self: A workaround is to put a space after the `#` in all my tasks blocks...

**Comment by damiankorcz** [comment]
Hi @claremacrae,
The tags in Source mode and when editing in Live Preview are rendered in 2 halves. The first containing the `#` (.cm-hashtag-begin) and the second containing the Tag name (.cm-hashtag-end). There are no classes added to the lines which are part of a codeblock that would allow me to make an exception based on it. So the only solution I have is to disable the Tag styling altogether in Source mode / Live Preview edit (I could do that as a toggle in Style Settings if that's something you'd be ok with), leave it as it is or as you noted add a space after the `#` which isn't ideal.

**Comment by claremacrae** [comment]
Hi @damiankorcz 

Thank you!

> The tags in Source mode and when editing in Live Preview are rendered in 2 halves. The first containing the `#` (`.cm-hashtag-begin`) and the second containing the Tag name (`.cm-hashtag-end`). 

Hah, that explains another oddity (for info, not for action)

In source mode, I use the Dynamic Highlights plugin to highlight tasks that are not done, in a sort-of yellow...

Since the new Prism, they now look like this - which I can definitely live with!

![image](https://user-images.githubusercontent.com/4840096/167306367-8fc130f6-670a-4aae-8ae3-dabbf73cee94.png)

> There are no classes added to the lines which are part of a codeblock that would allow me to make an exception based on it. So the only solution I have is to disable the Tag styling altogether in Source mode / Live Preview edit (I could do that as a toggle in Style Settings if that's something you'd be ok with), leave it as it is or as you noted add a space after the `#` which isn't ideal.

Since my workaround was quick  to apply and is already done, I would suggest not changing anything unless you hear from anyone else!

**Comment by damiankorcz** [comment]
This should now be addressed in Version 2.3.0. Sorry for taking this long on this one 😅

**Comment by claremacrae** [comment]
> This should now be addressed in Version 2.3.0. Sorry for taking this long on this one 😅

Not at all - thank you very much for working on it.
